charts/ratify/templates/store.yaml

```diff
@@ -19,7 +19,7 @@ spec:
     {{- if .Values.oras.authProviders.azureManagedIdentityEnabled }}
     authProvider:
       name: azureManagedIdentity
-      clientID: {{ .Values.azureManagedIdentity.clientId }}
+      clientID: {{ .Values.azureManagedIdentity.clientId | quote }}
```

Comment:
The e2e tests were running into an issue where the value is already quotes so this would double quote causing issues. I sent another PR to fix the issue in the configmap. Might need to remove this too?